cli/api/backup.go

```diff
@@ -29,13 +30,17 @@ func (a *api) Backup(dirpath string, excludes []string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
 	var path string
 	req := dao.BackupRequest{
 		Dirpath:              dirpath,
 		SnapshotSpacePercent: config.GetOptions().SnapshotSpacePercent,
 		Excludes:             excludes,
 	}
+
+	est := dao.BackupEstimate{}
```

Comment:
Can you move this call into the rpc package?  We shouldn't be creating new calls in the dao layer.